extensions/import/src/controllers/mainController.ts

```diff
@@ -31,11 +31,11 @@ export default class MainController extends ControllerBase {
 
 	public async activate(): Promise<boolean> {
 		return new Promise<boolean>(async (resolve) => {
+			await new ServiceClient(this._outputChannel).startService(this._context);
```

Comment:
Service startup can take a while - what about starting these both and then await'ing Promise.all? I haven't looked at initializeFlatFileProvider though so if that depends on the service being started then maybe you're just fixing a bug here anyways.

Reply:
Sure that could work